x-pack/plugin/security/src/test/java/org/elasticsearch/xpack/security/authc/jwt/JwtRealmTestCase.java

```diff
@@ -620,6 +708,7 @@ protected void doMultipleAuthcAuthzAndVerifySuccess(
                 final Exception authcFailed = new Exception("Authentication test failed.");
                 realmFailureExceptions.forEach(authcFailed::addSuppressed); // realm exceptions
                 authcFailed.addSuppressed(t); // final throwable (ex: assertThat)
+                LOGGER.error(t.getMessage());
```

Comment:
```suggestion
```

Reply:
`t` is added to authcFailed as a suppressed exception, and authcFails is logged at error level. Why log it again?

Reply:
Was a debug leftover, deleted it :P